src/config/links.ts

```diff
@@ -5,10 +5,11 @@ const INTERLAY_DISCORD_LINK = 'https://discord.gg/KgCYK3MKSf';
 const INTERLAY_GITHUB_LINK = 'https://github.com/interlay';
 const INTERLAY_TWITTER_LINK = 'https://twitter.com/interlayHQ';
 const INTERLAY_DOCS_LINK = 'https://docs.interlay.io';
-
-// TODO: placeholders
+const INTERLAY_CROWDLOAN_LINK = '';
```

Comment:
This was only being used for the claim nav item and I found it a bit more consistent and easier to understand to remove the link so that we can keep the logic for showing the nav item the same as the T&Cs.